packages/flutter_app/lib/pages/main/main_page.dart

```diff
@@ -17,28 +19,35 @@ class MainPage extends HookConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    ref.listen(networkConnectionStateProvider.future, (previous, next) async {
-      final messenger = ScaffoldMessenger.of(context);
-      final prevResults = await previous;
-      final nextResults = await next;
+    ref
+      ..listen(appLifecycleProvider, (prev, next) {
+        logger.finest('AppLifecycleState: prev: $prev, next: $next');
+      })
+      ..listen(localesProvider, (prev, next) {
+        logger.finest('Locales: prev: $prev, next: $next');
+      })
```

Comment:
listen` to `ref.listen` for `appLifecycleProvider` and `localesProvider` to make sure they are being monitored.